pkgs/top-level/python-aliases.nix

```diff
@@ -177,7 +178,7 @@ mapAliases {
   cozy = throw "cozy was removed because it was not actually https://pypi.org/project/Cozy/."; # added 2022-01-14
   criticality-score = throw "use pkgs.criticality-score instead"; # added 2024-12-31
   cryptacular = throw "cryptacular was removed, because it was disabled on all python version since 3.6 and last updated in 2021"; # Added 2024-05-13
-  cryptography_vectors = "cryptography_vectors is no longer exposed in python*Packages because it is used for testing cryptography only."; # Added 2022-03-23
+  cryptography_vectors = throw "'cryptography_vectors' has been renamed to/replaced by 'cryptography-vectors'" cryptography-vectors; # Converted to throw 2025-10-28
```

Comment:
This is wrong. Ths adds a second argument to `throw`, which is `cryptography-vectors`. We don't need that, only need the original string.